tar/bsdtar.c

```diff
@@ -1251,7 +1252,7 @@ configfile(struct bsdtar *bsdtar, const char *fname)
 		 * than failing to report an error if there really is a
 		 * configuration file.
 		 */
-		bsdtar_errc(bsdtar, 1, errno, "stat(%s)", fname);
+		bsdtar_errc(bsdtar, 1, errno, "configfile stat(%s)", fname);
```

Comment:
I'd keep the `stat(%s)` error message here but add a more user-friendly one for the fromcmdline && ENOENT case.